doc/example/example_visualization.ipynb

```diff
@@ -2,295 +2,574 @@
+   ],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
    "outputs": [],
    "source": [
-    "# Plot 3 axes objects, plotting\n",
-    "# - in the first window  the observable 'pS6_tot' for all simulation conditions\n",
-    "# - in the second window the observable 'pEGFR_tot' for all simulation conditions\n",
-    "# - in the third window  the observable 'pAkt_tot' for all simulation conditions\n",
-    "\n",
-    "observable_id_list = [[\"pS6_tot\"], [\"pEGFR_tot\"], [\"pAkt_tot\"]]\n",
-    "\n",
+    "petab.visualize.plot_problem(petab_problem)\n",
+    "plt.gcf().set_size_inches(10, 4)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Subplots by observable\n",
     "\n",
-    "ax = plot_without_vis_spec(\n",
-    "    condition_file_Fujita,\n",
-    "    observable_id_list,\n",
-    "    \"observable\",\n",
-    "    data_file_Fujita,\n",
-    "    plotted_noise=\"provided\",\n",
-    ")"
-   ]
+    "Next, to make the plot less crowded, we group measurements by observables by adding a second subplot and specifying which observables to plot in each via the `yValues` column:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
    "outputs": [],
    "source": [
-    "# Plot 2 axes objects, plotting\n",
-    "# - in the first window  the observable 'pS6_tot' for all simulation conditions\n",
-    "# - in the second window the observable 'pEGFR_tot' for all simulation conditions\n",
-    "# - in the third window  the observable 'pAkt_tot' for all simulation conditions\n",
-    "# while using the noise values which are saved in the PEtab files\n",
-    "\n",
-    "observable_id_list = [[\"pS6_tot\"], [\"pEGFR_tot\"]]\n",
+    "petab_problem.visualization_df = petab.get_visualization_df(\n",
+    "    example_dir_fujita / \"visuSpecs\" / \"Fujita_visuSpec_2.tsv\"\n",
+    ")\n",
+    "petab_problem.visualization_df"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "petab.visualize.plot_problem(petab_problem)\n",
+    "plt.gcf().set_size_inches(10, 4)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Subplots by dataset\n",
     "\n",
+    "We can also plot different datasets (as specified by the optional `datasetId` column in the measurement table) in separate subplots:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "petab_problem.visualization_df = petab.get_visualization_df(\n",
+    "    example_dir_fujita\n",
+    "    / \"visuSpecs\"\n",
+    "    / \"Fujita_visuSpec_individual_datasets.tsv\"\n",
```

Comment:
Let's remove the column `plotTypeSimulation`

Reply:
Remove from the file, or don't display?

Reply:
If the file is only for this part in the tutorial, remove from the file. I don't mind the `xValues`, just find the simulation specification misleading since there is no simulation.

Reply:
Got it. Done.